internal/sync/sync.go

```diff
@@ -66,6 +72,15 @@ func (s Sync) syncPaths(delete bool, local, remote string) error {
 	cmd.Stdin = os.Stdin
 	err := cmd.Run()
 	if err != nil {
+		if exitError, ok := err.(*exec.ExitError); ok {
+			if exitError.ExitCode() == ERR_RSYNC_INCOMPAT {
+				flog.Error("no compatible rsync present on remote machine")
+			} else if exitError.ExitCode() == ERR_RSYNC_STREAM {
+				flog.Error("error in rsync protocol datastream (no installed remote rsync?)")
+			} else {
+				flog.Error("sync: %v", err)
+			}
+		}
```

Comment:
Instead of logging the error here, I would prefer that the error is wrapped.

So something like:
```go
if exitError, ok := err.(*exec.ExitError); ok {
    if exitError.ExitCode() == ERR_RSYNC_INCOMPAT {
        return xerrors.Errorf("no compatible rsync present on remote machine: %w", err)
    }
    ...
```

Reply:
Ah, OK sure